site/docs/patterns/menu-button.mdx

```diff
@@ -0,0 +1,126 @@
+</ExampleContainer>
+
+### Layout
+
+- Use a [button](/salt/components/button) as the trigger for the main action. 
+- Use a [menu](/salt/components/menu) to display the selection of supplementary actions for users to choose from. 
+
+<Diagram
+  src="/img/patterns/menu-button/layout.png"
+  alt="layout"
+  border
+/>
+
+By default, the action button is aligned to the top left with the menu below. Depending on where the button is positioned within the available space and layout of the interface, the menu will dynamically adjust positions in order for its options to remain visible. 
+
+<Diagram
+  src="/img/patterns/menu-button/position.png"
+  alt="layout"
+  border
+/>
+
+<ImageSwitcher
+  images={[
+    { src: "/img/patterns/menu-button/screen-space-1.png", alt: "Menu button to the left of the screen with the menu open to the right" },
+    { src: "/img/patterns/menu-button/screen-space-2.png", alt: "Menu button to the right of the screen with the menu open to the left" },
+  ]}
+  label="Change screen space"
+/>
+
+## Menu button
+
+By default, menu button uses a primary button and is typically used when actions in the menu are related to the core functionality of an application. The button value is typically a string supported by a chevron icon, or can be an icon-only button, provided the icon is well-known and not likely to be misinterpreted. 
+
+<ImageSwitcher
+  images={[
+    { src: "/img/patterns/menu-button/in-layout.png", alt: "Menu button within a page" },
+    { src: "/img/patterns/menu-button/in-layout-open.png", alt: "Open menu button within a page" },
+  ]}
+  label="Open menu"
+/>
+
+<ExampleContainer type="positive" customPillText="Best practices">
+Consider your particular use case, the number and characteristics of menu options you need to display, your layout and available space – when determining the most appropriate button label.
+</ExampleContainer>
+
+## Overflow menu
+
+Overflow menus contain actions that affect a smaller type of content within an interface, such as within cards or data grids. This is typically represented by a three dot icon as the action button. This is useful when space is limited or the interface needs to remain uncluttered to reduce cognitive load. 
+
+<ImageSwitcher
+  images={[
+    { src: "/img/patterns/menu-button/overflow-menu.png", alt: "Overflow menu button within a page" },
+    { src: "/img/patterns/menu-button/overflow-menu-open.png", alt: "Open overflow menu button within a page" },
+  ]}
+  label="Open overflow menu"
+/>
+
+## Icon-only
+
+Icon-only menu buttons can be configured to contextually align to the product requirements, instead of being displayed as a chevron. See guidance for [button options](/salt/components/button) to select the most suitable button affordance for your application. 
```

Comment:
```suggestion
Icon only menu buttons can be configured to contextually align to the product requirements, instead of being displayed as a chevron. See guidance for [button options](/salt/components/button) to select the most suitable button affordance for your application. 
```